app/helpers/navigation_helper.rb

```diff
@@ -0,0 +1,100 @@
+module NavigationHelper
```

Comment:
this works, but what's the purpose of this reorg?

Reply:
The previous version would error when a user without :access_to_backend permission would try to access the admin side. This was because the admin_controller had overridden the method where the navbar items get generated, and thus the application navbar did not get the fields it expected. This way it doesn't matter which controller points someone to a layout anymore, which is more robust